### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by Flova**

### CODE REVIEW

**Comment by jaagut** [comment]
`rosdep_source.yml` I didn't know this file existed. Is this still needed? @timonegk

**Comment by Flova** [comment]
I think it was needed for the old CI

File: bitbots_docs/docs/manual/tutorials/vscode-ros2.rst [hunk 1]
```diff
@@ -70,8 +70,6 @@ An example vscode launch configuration and the corresponding python lauch file c
                         "base_link_frame": "base_link",
                         "odom_frame": "odom",
                         "map_frame": "map",
-                        "ball_frame": "ball",
-                        "ball_approach_frame": "ball_approach_frame",
                         "base_footprint_frame": "base_footprint"
                     }],
                 remappings=[("/head_motor_goals", "/DynamixelController/command")]
```
Comment (jaagut) [inline bitbots_docs/docs/manual/tutorials/vscode-ros2.rst:75]: Please update this entire example, as we don't have the head-behavior anymore.

Comment (Flova) [reply bitbots_docs/docs/manual/tutorials/vscode-ros2.rst:75]: Ahhh

Comment (Flova) [reply bitbots_docs/docs/manual/tutorials/vscode-ros2.rst:75]: I didn't reach the tools repo

**Comment by timonegk** [comment]
> `rosdep_source.yml` I didn't know this file existed. Is this still needed? @timonegk

It was needed for the old CI.

File: bitbots_docs/docs/manual/tutorials/vscode-ros2.rst (hunk 1 above)
Comment (jaagut) [reply bitbots_docs/docs/manual/tutorials/vscode-ros2.rst:75]: The entire file should be updated with our new configuration… Let's do that after the cleanup. I have created this issue: https://github.com/bit-bots/bitbots_tools/issues/160

**Review by jaagut — DISMISSED** [review]

**Review by jaagut — APPROVED** [review]

**Pull request merged**